data_store/transformation/pathfinders/pf_transform_r1.py

```diff
@@ -123,15 +123,12 @@ def transform(df_dict: dict[str, pd.DataFrame], reporting_round: int) -> dict[st
 
 def _submission_ref(
```

Comment:
Style thing but now we've removed the second arg, this would look neater as a one-line signature. Slight shame that our current auto-formatters don't handle this sort of thing

Reply:
This is specifically left to the user to decide. If you add a trailing comma, it will be split over multiple lines. If you don't, then it will collapse it down as far as possible.

I think generally multiline is 'better', because the git diff is simplified when adding/removing lines. But it's so so minor that I don't pay attention to it any more really.